### PR TITLE
Review Changes : Updated App with a lots of features

### PR DESCRIPTION
**Pull request by Zaraki596**
Many changes and optimizations with updated new and optimized design.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,14 +1,36 @@
+#built application files
+*.apk
```
Comment (Zaraki596) [inline .gitignore:2]: I guess Remove this thing for the apk to be included in the file

Comment (PatilShreyas) [reply .gitignore:2]: No. APK should not be included in the repository. It'll available from `releases`.

Comment (Zaraki596) [reply .gitignore:2]: One more thing I forget to update the styles to NoActionBar please update that after merging

Comment (PatilShreyas) [reply .gitignore:2]: Yeah okay!

**Comment by PatilShreyas** [comment]
Thanks, @Zaraki596. Now it's perfect! Now going to merge it..

**Comment by Zaraki596** [comment]
Welcome @PatilShreyas , Love to work on it again, Hope to add some more features

**Pull request merged**